src/VisualStudio/Core/Impl/RoslynVisualStudioWorkspace.cs

```diff
@@ -107,6 +107,7 @@ internal override IInvisibleEditor OpenInvisibleEditor(DocumentId documentId)
 
             var symbolId = SymbolKey.Create(symbol, cancellationToken);
             var currentCompilation = currentProject.GetCompilationAsync(cancellationToken).WaitAndGetResult(cancellationToken);
+            Contract.ThrowIfNull(currentCompilation);
```

Comment:
We have an extension method GetRequiredCompilationAsync() which throws if it's not supported.